notebooks/official/prediction/pytorch_image_classification_with_prebuilt_serving_containers.ipynb

```diff
@@ -0,0 +1,883 @@
+{
```

Comment:
<p>Because the user is only issuing a single prediction request, prefer singular terms:</p><p> </p><h3>Get an online prediction</h3><p> </p><p> </p><p>You send a <code>predict</code> request with a single encoded image to the <code>endpoint</code>.</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1592/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='48'/>